renovate.json

```diff
@@ -3,5 +3,11 @@
   "labels": ["renovate"],
   "automerge": true,
   "automergeType": "branch",
-  "dependencyDashboard": true
+  "dependencyDashboard": true,
+  "packageRules": [
+    {
+      "updateTypes": ["major"],
+      "automergeType": "pr"
+    }
+  ]
```

Comment:
I'm not extremely confident in the test coverage in titus, therefore let's have Renovate create PRs for major bumps rather than silently merging them